### DISCUSSION TITLE
Stop waydroid from auto generating desktop launchers for every app installed

### DISCUSSION THREAD

**Discussion by adenlall** [waydroid/waydroid#1328] [Ideas]
Hi,
I'm just wondering if there any way to config waydroid to not generate .desktop files for every app installed. I can block it doing so on KDE but is there any method for GNOME, I remove all .desktop files related to waydroid but restart the session cause regenerating theme all.

**Comment by DoughMucker** [comment]
waydroid.sample.android.filename.desktop files are stored in .local/share/appications add NoDisplay=true to the apps you want to hide:

[Desktop Entry]
otherstuff
NoDisplay=true

**Comment by FaintShadow** [comment]
> Hi, I'm just wondering if there any way to config waydroid to not generate .desktop files for every app installed. I can block it doing so on KDE but is there any method for GNOME, I remove all .desktop files related to waydroid but restart the session cause regenerating theme all.

I have posted about it #1687 is you want, also your question seems to be more of an issue than a discussion.
